storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/PostgresQueryTemplater.java

```diff
@@ -72,13 +72,13 @@ public PostgresQueryTemplater(JdbcConnectionProvider connectionProvider) {
 
   @Override
   protected List<String> createLoadEntityQuery(
-      String destinationTable, String temporaryTable, File filePath) {
+      String destinationTable, String temporaryTable, URI fileUri) {
```

Comment:
Use string instead URI or File unless feast master has it in the interface

Reply:
ok, I think Feast is using string in the bigquery.